### DISCUSSION TITLE
Material Design based Time picker

### DISCUSSION THREAD

**Discussion by SudhagarMurugesan** [ng-matero/extensions#129] [Ideas]
Hi @nzbin, 
Bro can you add  use entered date and time input in time picker component on your ng-matero/extensions library.

![image](https://user-images.githubusercontent.com/87130720/187186434-119239cd-780e-4438-91c6-dae6a639cdc0.png)

dial based selection is really good one.
and it would more even simpler if there is an additional option to choose time via user input as well dial based selection just like in the [Material IO](https://material.io/components/time-pickers). 

Thankyou.

**Comment by nzbin** [comment]
Thanks for your advice. I'm very busy now and I haven't enough time to add more details. Maybe I will improve the timepicker in the future.

**Comment by nzbin** [comment]
The new version 14.6.0 has added the manual time input, please check the release info.
https://github.com/ng-matero/extensions/releases/tag/v14.6.0
